File: api/BaseApi.js
import fetch from 'node-fetch'

class BaseApi {
    
    static async postRequest(url, methodUrl, headers, data){
        const response = await fetch(url + methodUrl, {
            method: "POST",
            headers: headers,
            body: data
        })
        return response
    }

    static async getRequest(url, methodUrl){
        const response = await fetch(url + methodUrl)
        return response
    }
}
export default BaseApi

File: Api/BaseApi.py
import requests


class BaseApi:

    @staticmethod
    def post_request(url, method_url, data, headers):
        response = requests.post(url=(url + method_url), data=data, headers=headers)
        return response

    @staticmethod
    def get_request(url, method_url):
        response = requests.get(url=(url + method_url))
        return response
